instrumentation/opentelemetry-instrumentation-logging/src/opentelemetry/instrumentation/logging/__init__.py

```diff
@@ -61,8 +66,10 @@ def log_hook(span: Span, record: LogRecord):
 
     Args:
         tracer_provider: Tracer provider instance that can be used to fetch a tracer.
-        set_logging_format: When set to True, it calls logging.basicConfig() and sets a logging format.
-        logging_format: Accepts a string and sets it as the logging format when set_logging_format
+        set_logging_format: When set to True, it calls logging.basicConfig() and sets
+        a logging format.
```

Comment:
I think you need to indent if you continue the same field on a new line

Reply:
@xrmx, fixed